{"name": "test_list_users_invalid_params", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 in [400, 422]\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_batch_query_users.TestBatchQueryUsers object at 0x0000012A594A5940>\n\n    @allure.story(\"List users with invalid parameters\")\n    def test_list_users_invalid_params(self):\n        api = UserAPI()\n        res = api.get_users(page=\"abc\", size=-1)\n>       assert res.status_code in [400, 422]\nE       assert 404 in [400, 422]\nE        +  where 404 = <Response [404]>.status_code\n\ntestcases\\test_batch_query_users.py:33: AssertionError"}, "start": 1760265817175, "stop": 1760265818888, "uuid": "c2c5a3ee-7e58-4e1b-8bda-523f08f54945", "historyId": "1039087c4d177e1f42dded47e478f1e4", "testCaseId": "1039087c4d177e1f42dded47e478f1e4", "fullName": "testcases.test_batch_query_users.TestBatchQueryUsers#test_list_users_invalid_params", "labels": [{"name": "story", "value": "List users with invalid parameters"}, {"name": "feature", "value": "Batch Query Users API"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_batch_query_users"}, {"name": "subSuite", "value": "TestBatchQueryUsers"}, {"name": "host", "value": "DESKTOP-AC1JF6R"}, {"name": "thread", "value": "2260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_batch_query_users"}], "titlePath": ["testcases", "test_batch_query_users.py", "TestBatchQueryUsers"]}